README.org

```diff
@@ -1,17 +1,46 @@
 #+TITLE: Ren'Py mode
 
-PyTom's old major mode for Ren'Py, previously maintained by treymerkley, now maintained by me.
+A major mode for editing Ren'Py files.
 
 ** Installation
 
   Either install within Emacs directly from MELPA or:
+
   * Download/clone this repo to your desired location.
+
   * In Emacs, use the command =M-x package-install-file= and point it to the renpy.el file location
-    * This will install it to your =~/.emacs.d/elpa/= directory (or your custom location if you've manually set a different =package-user-dir= in Emacs)
+
+    * This will install it to your =~/.emacs.d/elpa/= directory (or your custom location
+      if you've manually set a different =package-user-dir= in Emacs)
```

Comment:
Given that the instructions to install from MELPA do not say where the files get installed, it is probably best to also omit that from the instructions which install from a file.